AirCasting/SDSync/ViewsFlow/SDSyncUnplugAB/UnplugABView.swift

```diff
@@ -0,0 +1,75 @@
+// Created by Lunar on 08/12/2021.
+//
+
+import AirCastingStyling
+import SwiftUI
+
+struct UnplugABView<VM: UnplugABViewModel>: View {
+    @StateObject var viewModel: VM
+    @Binding var creatingSessionFlowContinues: Bool
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 40) {
+            ProgressView(value: 0.3)
+            Spacer()
+            HStack() {
+                Spacer()
+                unplugImage
+                Spacer()
+            }
+            Spacer()
+            VStack(alignment: .leading, spacing: 15) {
+                titleLabel
+                messageLabel
+            }
+            continueButton
+            Spacer()
+        }
+        .background(navigationLink)
+        .padding()
+    }
+}
+
+extension UnplugABView {
+    
+    var unplugImage: some View {
+        Image("airbeam")
+            .resizable()
+            .aspectRatio(contentMode: .fit)
+    }
+    
+    var titleLabel: some View {
+        Text(Strings.UnplugAirbeamView.title)
+            .font(Fonts.boldTitle3)
+            .foregroundColor(.accentColor)
+    }
+    
+    var messageLabel: some View {
+        Text(Strings.UnplugAirbeamView.message)
+            .font(Fonts.regularHeading1)
+            .foregroundColor(.aircastingGray)
+    }
+    
+    var continueButton: some View {
+        Button {
+            viewModel.presentNextScreen = true
```

Comment:
As I remember we have used in others SD sync views func in ViewModel to do it, and then just call that function here. Please do it here too 😄

Reply:
+1